### ISSUE TITLE
FCOS3D support

### ISSUE THREAD

**Issue by topys21c** [open-mmlab/mmdetection3d#247]
Hi, thank you for sharing a nice repository.

I found the FCOS3D results in nuScenes leaderboard.
When will the FCOS3D baseline be updated?

I would like to know if there is an upcoming schedule. 

Thank you.

**Comment by Tai-Wang** [comment]
Thanks for your interests.

Yes, it has been our plan. Currently we still have some issues related to that baseline to deal with, and there are other items on our list to support with higher priority. So I hope it should be done in February/March.

**Comment by topys21c** [comment]
Thank you for your quick reply.

Is it possible to share the code before you merge it into the master branch even there are some issues?
I understand that there will be higher priority lists, and I understand that even if you cannot share it.

Hope you have a nice holiday. Merry Christmas!

**Comment by Tai-Wang** [comment]
I am sorry that it may be inconvenient now even though I am willing to share it. Hope you can understand.
Considering your strong interests and maybe also urgent needs from community, I would try my best to push it. Thanks for your patience in advance.

Finally, also the same best wishes to you. Merry Christmas :)

**Comment by topys21c** [comment]
Hi Tai-Wang,
I am trying to implement FCOS and CenterNet on nuscenes dataset but failed to reproduce the desired performance.
Can you kindly share the training hyperparameters of your experiments?
total epoch, learning rate, learning schedule, learning rate decay will be very helpful for me.
Thank you!

**Comment by Tai-Wang** [comment]
There is nothing special in terms of these training hyperparameters. I just use 12 epochs with mmdet schedule_1x (simliar with this [schedule](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/_base_/schedules/mmdet_schedule_1x.py)). I think more tricky things are located in many implementation details of the overall framework. I also spent much time tuning and debugging the transferred method.

**Comment by topys21c** [comment]
Thank you for your advice.
I was trying ADAM with a lower learning rate, and will refer to your recommendation.
I will look forward to the code release. Thank you!

**Comment by filaPro** [comment]
Hi @Tai-Wang ,
Can you please briefly describe your idea behind aggregating predicted boxes from all 6 cameras? Is it just NMS with 3d IoU or something more specific? Thanks in advance.

**Comment by Tai-Wang** [comment]
For predictions, I just use NMS with 3D IoU to remove redundant predictions appeared in the overlap of adjacent images and get a very limited performance gain. Actually, it is hardly important for inaccurate predictions from my basic experiments, which is different from when aggregating accurate ground truth boxes.

**Comment by vobecant** [comment]
Are there any news regarding releasing the FCOS model trained for 3D detection from RGB images? Thank you.
